openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/SchemaTablesTab.tsx

```diff
@@ -137,38 +136,36 @@ function SchemaTablesTab({
         </Row>
       </Col>
       <Col span={24}>
-        {tableDataLoading ? (
-          <Loader />
-        ) : isEmpty(tableData) && !showDeletedTables ? (
-          <ErrorPlaceHolder
-            className="mt-0-important"
-            type={ERROR_PLACEHOLDER_TYPE.NO_DATA}
-          />
-        ) : (
-          <TableAntd
-            bordered
-            columns={tableColumn}
-            data-testid="databaseSchema-tables"
-            dataSource={tableData.data}
-            locale={{
-              emptyText: <ErrorPlaceHolder />,
-            }}
-            pagination={false}
-            rowKey="id"
-            size="small"
-          />
-        )}
-
-        {tableData.paging.total > PAGE_SIZE && tableData.data.length > 0 && (
+        <TableAntd
```

Comment:
```suggestion
        <Table
```

Reply:
we have a clashing name with Type Table.